app/org/sagebionetworks/bridge/models/substudies/AccountSubstudyId.java

```diff
@@ -0,0 +1,56 @@
+package org.sagebionetworks.bridge.models.substudies;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public final class AccountSubstudyId implements Serializable {
+    private static final long serialVersionUID = -2741510011134968409L;
```

Comment:
Minor: Remove serialVersionUID and suppress warning.

Reply:
Removed